test/integration/route.spec.js

```diff
@@ -256,9 +256,9 @@ describe('Routing feature server requests', () => {
           res.body.features[0].attributes.max_2.should.equal(57611)
           res.body.features[0].attributes.count_2.should.equal(75343)
           res.body.fields.length.should.equal(3)
-          res.body.fields[0].type.should.equal('esriFieldTypeInteger')
-          res.body.fields[1].type.should.equal('esriFieldTypeInteger')
-          res.body.fields[2].type.should.equal('esriFieldTypeInteger')
+          res.body.fields[0].type.should.equal('esriFieldTypeDouble')
+          res.body.fields[1].type.should.equal('esriFieldTypeDouble')
+          res.body.fields[2].type.should.equal('esriFieldTypeDouble')
```

Comment:
Assertions altered here after I confirmed this is the expected behavior with ArcGIS Server.